ui/src/controls/ToggleDarkModeButton.svelte

```diff
@@ -0,0 +1,32 @@
+<svelte:head>
+{#if stored == "dark"}
+<link rel='stylesheet' href='darkmode.css' id="dark-mode-css">
+{/if}
+</svelte:head>
+
+<script>
+import { createEventDispatcher } from 'svelte';
+const dispatch = createEventDispatcher();
+var stored = localStorage.getItem("dark-mode-storage");
+
+if (!stored) {
+  if (window.matchMedia("(prefers-color-scheme: dark)").matches ) {
+    stored = "dark";
+  } else {
+    stored = "light";
+  }
+}
+
+function switchTheme() {
+    if (stored == "light") {
+        stored = "dark";
+    } else {
+        stored = "light";
+      }
```

Comment:
Please use consistent indentation as specified in the editorconfig (two spaces).

Reply:
Also, somewhere here we should set the localStorage

Reply:
Indentation fixed in https://github.com/liwde/blinddichten/pull/1/commits/4a501f63306436014bb80975c50d0590f93fcde8, isnt' localStorage set in lines 8 to 16?

Reply:
You're just reading from the localStorage, never writing. The variable would better be named "theme"...